Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[workspace]
+resolver = "2"
+
+members = ["crates/bcr-wallet-core"]
+
+[workspace.dependencies]
+bcr-ebill-core = { git = "https://github.com/BitcreditProtocol/Bitcredit-Core.git", tag = "v0.3.11" }
+bcr-ebill-api = { git = "https://github.com/BitcreditProtocol/Bitcredit-Core.git", tag = "v0.3.11" }
+bcr-wdc-webapi = { git = "https://github.com/BitcreditProtocol/Wildcat" }
+anyhow = { version = "1" }
+async-trait = { version = "0.1" }
+axum = { version = "0.8" }
+axum-test = { version = "17" }
```

Comment:
axum? :thinking: 

I guess this is just a collection of crates we use generally to start with and select out later, right?